.travis.yml

```diff
@@ -26,8 +26,8 @@ before_script:
 - docker run -d --net=host --shm-size=2g selenium/standalone-chrome:3.141.59-oxygen
 - export DBUS_SESSION_BUS_ADDRESS=/dev/null
 - export DISPLAY=:99.0
+- sleep 3
```

Comment:
@PeterNgTr for what to we wait here?

Reply:
wait for selenium is ready otherwise it will be failed.